getting-started/blazor/your-first-chart-using-blazor.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Your First Chart in Blazor using FusionCharts | FusionCharts
+description: This article outlines the steps to be executed for creating your first chart using the blazor-fusioncharts component.
+heading: Create a Chart in Blazor using FusionCharts
+---
+
+FusionCharts is a JavaScript charting library that enables you to create interactive charts, gauges, maps, and dashboards in JavaScript. The `blazor-fusioncharts` component lets you easily elevate data visualization to any Blazor project. On this page, we'll see how to install FusionCharts and render a chart using the `blazor-fusioncharts` component.
```

Comment:
just curious, from where did we get the term 'blazor-fusioncharts'? AFAIK the component/package name on nuget is BlazorFusionCharts.